.github/workflows/deploy_to_azure.yml

```diff
@@ -0,0 +1,48 @@
+name: Deploy to stage and prod
```

Comment:
Strange that this change is coming from `master` when merging to `develop`, don't you agree, @NullableInt?

Reply:
I don't quite get what you are asking for here.

Reply:
Mystery solved, #839 was never merged to `develop`.